src/app/prescription/medication-request-form/medication-form/medication-form.component.css

```diff
@@ -24,3 +24,16 @@ button.mat-stroked-button {
   font-weight: 400;
   line-height: 1.125;
 }
+
+.container-ingredient-amount-form {
```

Comment:
Would call it something like "inline-container" if that makes sense ? (this way its more re-usable if we ever wanted it in a common css ? Not really usefull in this context but more of a good practice)